packages/devtools_app/lib/src/network/flutter/network_screen.dart

```diff
@@ -0,0 +1,312 @@
+      return const TextStyle(color: Colors.greenAccent);
+    } else if (statusInt >= 300 && statusInt < 400) {
+      return const TextStyle(color: Colors.yellowAccent);
+    }
+    return const TextStyle();
+  }
+
+  @override
+  DataRow getRow(int index) {
+    final data = _data[index];
+    final numFormat = NumberFormat.decimalPattern();
+    final status = (data.status == null) ? 'N/A' : data.status.toString();
+    final requestTime = DateFormat.Hms().add_yMd().format(data.requestTime);
+    final TextStyle statusColor = _getStatusColor(data.status);
+
+    final durationMs = (data.durationMs == null)
+        ? 'In Progress'
+        : numFormat.format(data.durationMs);
+
+    return DataRow.byIndex(
+        index: index,
+        cells: <DataCell>[
+          DataCell(Text(data.name)),
+          DataCell(Text(
+            data.method,
+            style: const TextStyle(fontWeight: FontWeight.bold),
+          )),
+          DataCell(Text(status, style: statusColor)),
+          DataCell(Text(durationMs)),
+          DataCell(Text(requestTime)),
+        ],
+        selected: data.selected,
+        onSelectChanged: (bool selected) {
+          if (data != _currentSelection.value) {
+            _currentSelection.value?.selected = false;
+            data.selected = true;
+            _currentSelection.value = data;
+          } else if (data == _currentSelection.value) {
+            _currentSelection.value = null;
+            data.selected = false;
+          }
+          notifyListeners();
+        });
+  }
+
+  @override
+  int get rowCount => _data?.length ?? 0;
+
+  void clearSelection() => _currentSelection.value = null;
+
+  ValueListenable<HttpRequestData> get currentSelectionListenable =>
+      _currentSelection;
+  final _currentSelection = ValueNotifier<HttpRequestData>(null);
+
+  @override
+  bool get isRowCountApproximate => false;
+
+  @override
+  int get selectedRowCount => 0;
+}
+
+class NetworkScreen extends Screen {
+  const NetworkScreen() : super();
+
+  @override
+  Widget build(BuildContext context) => const NetworkScreenBody();
+
+  @override
+  Widget buildTab(BuildContext context) {
+    return const Tab(
+      text: 'Network',
+      icon: Icon(Icons.network_check),
+    );
+  }
+}
+
+class NetworkScreenBody extends StatefulWidget {
+  const NetworkScreenBody();
+
+  @override
+  State<StatefulWidget> createState() => NetworkScreenBodyState();
+}
+
+class NetworkScreenBodyState extends State<NetworkScreenBody> {
+  static bool _sortAscending = false;
+  static int _sortColumnIndex;
+
+  final networkController = NetworkController();
+  final dataTableSource = HttpRequestDataTableSource();
+
+  void _onSort(Function getField, int columnIndex, bool ascending) {
+    dataTableSource._sort(
+      getField,
+      columnIndex,
+      ascending,
+    );
+    setState(() {
+      _sortAscending = ascending;
+      _sortColumnIndex = columnIndex;
+    });
+  }
+
+  static const _headerTextStyle = TextStyle(
```

Comment:
All colors needed to be themed so they will work with both the dark and light theme. When possible try use a color and better yet font size and font weight from the material theme.

Reply:
I think I've covered all these instances where I was creating custom text styles. If you see any I've missed, please let me know.